metadata-ingestion/src/datahub/ingestion/source/powerbi/proxy.py

```diff
@@ -447,6 +449,54 @@ def get_dataset(self, workspace_id: str, dataset_id: str) -> Any:
             tables=[],
         )
 
+    def get_dataset_schema(self, dataset: PowerBIDataset) -> Dict[str, List[str]]:
```

Comment:
The origin of this PR is the fact that when using ODBC datasources in PowerBI the DataHub ingestion completely skips ingestion of datasets using ODBC. This is mainly because the workspace scan result dict has `tables` as empty list. There's no column information present at the payload either.

In order to get anything in the datasets visible, I implemented the DAX query to achieve that. I suspect that using ODBC isn't that popular in the end. We're currently forced to use ODBC datasources in PowerBI.

This is `datasect_dict` contents used in https://github.com/datahub-project/datahub/pull/6440/files#diff-72724d37501e21166f538e16168db8dbec7a9b17f93e4681443335e9136e158aR920 (I redacted ID's and other sensitive data from payload, the point is that the tables list is empty).
```
{'configuredBy': 'xxxxx',
 'configuredById': 'xxxxx',
 'contentProviderType': 'PbixInImportMode',
 'createdDate': '2022-12-01T11:07:40.787',
 'datasourceUsages': [{'datasourceInstanceId': 'xxxxxxx'}],
 'description': '.... ',
 'id': 'xxxxx',
 'isEffectiveIdentityRequired': False,
 'isEffectiveIdentityRolesRequired': False,
 'name': 'xxxx',
 'tables': [],
 'targetStorageMode': 'PremiumFiles',
 'users': [.....]}

```